yarn.lock

```diff
@@ -1130,7 +1130,7 @@ chalk@^2.4.2:
     escape-string-regexp "^1.0.5"
     supports-color "^5.3.0"
 
-chalk@^4.0.0, chalk@^4.1.2:
+chalk@^4.0.0:
```

Comment:
What else is still pulling `chalk` in?

Reply:
Chalk is used in devDependencies of jscodeshift, like jest and eslint.

I've created feature requests with those repos, documented as a follow-up in https://github.com/facebook/jscodeshift/issues/600#issuecomment-2227665223